parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -275,12 +279,14 @@ try
                     -IdentitySystem "{{ GetIdentitySystem }}"
             }
 
-        } elseif ($global:NetworkPlugin -eq "kubenet") {
+            New-ExternalHnsNetwork

```

Comment:
just to clarify - is `New-ExternalHnsNetwork` not required for kubenet?

Reply:
It isn't being created for kubenet today

Reply:
So i assume it isn't

Reply:
The removal is in the code block for if ($NetworkPlugin -eq "azure") { }

Reply:
Maybe @pradipd  or @dineshgovindasamy  can comment on what scenarios we should create the ext HNS network?

Reply:
It should be created if you want to prevent that brief network disconnect on reboot

Reply:
I updated to create the network for all network plugings.